_includes/scenarios/integrate/create-own-identityattribute.md

```diff
@@ -0,0 +1,111 @@
+This guide explains the end to end flow of creating an [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) for your own Connector. Before you start, you should check out our IdentityAttribute introduction first.
+
+<!--- [IdentityAttribute introduction]({_docs_integrate/identityattribute-introduction.md }) --->
+
+## Send Request
+
+To create an IdentityAttribute as a user for your own Connector, you need to send a Request by executing `POST /api/v2/Attributes` with an IdentityAttribute JSON payload:
```

Comment:
Maybe a bit confusing, since "user" usually is associated with the App and not the Connector. Maybe write maintainer/developer/integrator or just omit it.